modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -30,10 +25,14 @@ An API fragment is a RAML document that has a version and an identifier, but is
 
 == Procedure
 
-. On the *Projects* page in Design Center, click *Create* and select *Create Fragment*.
-. In the *New Fragment* dialog, type a name for the project. You can change the name (but not the extension) later, if you want to.
-. Select the type of API fragment that you want to create and click *Create Fragment*.
+. On the *Projects* page in Design Center, click *Create new*.
+. Select *New Fragment*.
+. In the *New Fragment* dialog, type a name for the Fragment project. 
 +
+You can change the name (but not the extension) later, if you want to.
+
+. Select the type of API fragment that you want to create and click *Create Fragment*.
```

Comment:
You can change the fragment type at any point. I don't know what they mean by extension though.

Reply:
Changed to: "You can change the name or fragment type later, if necessary."

Not sure about "extension" either. Let me know if you think the above change doesn't reflect the behavior accurately.